torch/lib/TH/generic/THTensorMath.c

```diff
@@ -3127,18 +3127,25 @@ accreal THTensor_(meanall)(THTensor *tensor)
   return THTensor_(sumall)(tensor)/THTensor_(nElement)(tensor);
 }
 
-accreal THTensor_(varall)(THTensor *tensor)
+accreal THTensor_(varall)(THTensor *tensor, int flag)
 {
   accreal mean = THTensor_(meanall)(tensor);
   accreal sum = 0;
   TH_TENSOR_APPLY(real, tensor, sum += (*tensor_data - mean)*(*tensor_data - mean););
-  sum /= (THTensor_(nElement)(tensor)-1);
+  if (flag)
+  {
+    sum /= THTensor_(nElement)(tensor);
+  }
+  else
+  {
+    sum /= (THTensor_(nElement)(tensor)-1);
+  }
```

Comment:
Since I'm at it, I will change all occurrences of `flag` in `var`, `std`, `varall`, `stdall` to `biased` (1 yields the biased estimate) for clarity.